soc/arm/nordic_nrf/CMakeLists.txt

```diff
@@ -25,4 +25,8 @@ if(CONFIG_BUILD_WITH_TFM)
   set_property(TARGET zephyr_property_target
     APPEND PROPERTY TFM_CMAKE_OPTIONS -DZEPHYR_BASE=${ZEPHYR_BASE}
     )
+
+  set_property(TARGET zephyr_property_target
+    APPEND PROPERTY TFM_CMAKE_OPTIONS -DNRF_NS_STORAGE=${CONFIG_TFM_NRF_NS_STORAGE_PARTITION}
+    )
```

Comment:
Do we have a convention for translating vanilla TF-M CMake options to Kconfig?

Perhaps we should have this code next to the other CMake code that does the same thing.

And also follow the naming convention.

E.g. if the TF-M option ends with STORAGE, then I assume the Kconfig option should not end with PARTITION?